README.md

```diff
@@ -268,7 +268,7 @@ git checkout rel/x.x
 
 ### Build IoTDB from source
 
-Under the root path of iotdb:
+Under the iotdb/iotdb-client path:
```

Comment:
Building IoTDB is not related with iotdb-client....

Reply:
fixed